site/documentation/content/api/management/device-registry-v1.yaml

```diff
@@ -36,7 +36,7 @@ info:
    license:
       name: EPL-2.0
       url: https://www.eclipse.org/legal/epl-2.0/
-   version: 1.2.0
+   version: 1.3.0
```

Comment:
The API is already at 1.3.0 on master ...

Reply:
fixed.